tools/acle_template.tex

```diff
@@ -24,16 +23,19 @@
 \usepackage[defaultsans,scale=0.95]{lato}
 \usepackage{inconsolata}
 \renewcommand*\familydefault{\sfdefault}
-
-$if(geometry)$
-\usepackage[$for(geometry)$$geometry$$sep$,$endfor$]{geometry}
-$endif$
-
+\usepackage{graphicx}
+\usepackage[$if(landscape)$landscape,$endif$top=3.7cm,bottom=2.7cm,left=3cm,right=2cm,headsep=1.5cm,footskip=.5cm]{geometry}
 \usepackage{lastpage}
+\usepackage{dblfloatfix}
+\usepackage{subcaption}
+
+\definecolor{grey}{rgb}{0.5,0.5,0.5}
+\definecolor{lightgrey}{rgb}{0.83,0.83,0.83}
+\definecolor{armblue}{HTML}{0091BD}
```

Comment:
Could you please mention that these are official arm colors and reference the place where you got them from? (I imagine from here https://www.arm.com/company/policies/trademarks/guidelines-corporate-logo, although I cannot find references to lightgray and grey? Or are these colors that @sallyarmneale told you to use?

Moreover, I see that the color `darkgray` is used below. Is this a custom definition or a color name predefined by `xcolor`? I think we should custom define all out colors as `armgray` `armlightgray` `armblue` and `armdarkgray` (or `armblack`  as defined in https://www.arm.com/company/policies/trademarks/guidelines-corporate-logo?) so that we will not clash with any `xcolor` names.

Reply:
Those colors are used in the grid image in the title page. I copied them from the source code for the grid image. darkgray is a predefined xcolor colour.